ipaserver/install/bindinstance.py

```diff
@@ -682,7 +682,7 @@ def setup(self, fqdn, ip_addresses, realm_name, domain_name, forwarders,
         self.forward_policy = forward_policy
         self.reverse_zones = reverse_zones
 
-        if zonemgr is not None:
+        if zonemgr is None:
```

Comment:
I would rather write `if not zonemgr:` as this would also cover the case where the user provided an empty string with `--zonemgr ""`.

Reply:
OK. I also changed another occurrence in a method above.